.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+    "env": {
+        "browser": true,
+        "es2017": true,
+        "greasemonkey": true
+    },
+    "extends": ["eslint:recommended", "plugin:compat/recommended"],
+    "parserOptions": {
+        "ecmaVersion": 9
+    },
+    "rules": {
+        "no-unused-vars": "off",
+        "no-undef": "off",
```

Comment:
Of course, if one wants to find unused/undef stuff too, I suppose one can just run eslint agains the compiled script? Sure one would need to look up the line number too to find the correct module, so it's slower, but it should be possible.

Reply:
Hm, good point. Right now the config applies equally to both commands, so maybe I can see if I can turn these back on when it runs on the compiled script.

Reply:
Worst case, that can be done in a text editor on demand, so not important for this PR